### PR TITLE
kong: bring content up to date

### PR DESCRIPTION
**Pull request by hanshuebner**

### CODE REVIEW

File: kong/content.md [hunk 1]
```diff
@@ -1,174 +1,20 @@
 # What is Kong?
 
-Kong is a scalable, open source API Platform (also known as an API Gateway or API Middleware). Kong was originally built by [Kong Inc.](https://konghq.com) (formerly known as Mashape) to secure, manage, and extend over 15,000 Microservices for its API Marketplace, which generates billions of requests per month.
+Kong Gateway is the world’s most adopted API gateway. Lightweight,
+fast, and flexible, this open source gateway is built for hybrid and
+multi-cloud and optimized for microservices and distributed
+architectures.
```
Comment (kikito) [inline kong/content.md:6]:
```suggestion
Kong Gateway is the world’s most adopted API gateway. Lightweight, fast, and flexible, this open source gateway is built for hybrid and multi-cloud and optimized for microservices and distributed architectures.
```

File: kong/content.md [hunk 2]
```diff
@@ -1,174 +1,20 @@
...
 
-Under active development, Kong is now used in production at hundreds of organizations from startups, to large enterprises and governments including: The New York Times, Expedia, Healthcare.gov, The Guardian, Condè Nast, The University of Auckland, Ferrari, Rakuten, Cisco, SkyScanner, Yahoo! Japan, Giphy and so on.
+Built on this open source DNA, Kong’s unified cloud API platform helps
+organizations ranging from startups to Fortune 500 enterprises unleash
+developer productivity, build securely, and accelerate time to market.
```
Comment (kikito) [inline kong/content.md:10]:
```suggestion
Built on this open source DNA, Kong’s unified cloud API platform helps organizations ranging from startups to Fortune 500 enterprises unleash developer productivity, build securely, and accelerate time to market.
```

File: kong/content.md [hunk 3]
```diff
@@ -1,174 +1,20 @@
...
 
-Kong's official documentation can be found at [docs.konghq.com](https://docs.konghq.com/).
 
-# How to use this image without a Database
+Kong's official documentation can be found at
+[docs.konghq.com](https://docs.konghq.com/).
```
Comment (kikito) [inline kong/content.md:14]:
```suggestion
Kong's official documentation can be found at [docs.konghq.com](https://docs.konghq.com/).
```

File: kong/content.md [hunk 4]
```diff
@@ -1,174 +1,20 @@
...
-    -p 8444:8444 \
-    %%IMAGE%%
-```
-
-# Kubernetes Ingress
-
-Among the many deployment options [available](https://konghq.com/install), Kong also offers a [Kubernetes Ingress Controller](https://github.com/Kong/kubernetes-ingress-controller) ready to use in your K8s environment.
+Please refer to the
+[installation section](https://docs.konghq.com/gateway/latest/install/docker/#main)
+on our documentation website to learn how to use this image.
```
Comment (kikito) [inline kong/content.md:20]:
```suggestion
Please refer to the [installation section](https://docs.konghq.com/gateway/latest/install/docker/#main) on our documentation website to learn how to use this image.
```

**Review by kikito — COMMENTED** [review]
It seems that this repo's markdown linter (markdownfmt) wants paragraphs to be in a single line long line instead of multiple shorter ones. Suggestions added.

**Comment by hanshuebner** [comment]
Hello, is there any way how we can get this merged?

**Comment by yosifkit** [comment]
🙇 Apologies. This slipped through the cracks.

Close/reopen to get github actions to run again.

**Comment by yosifkit** [comment]
Squash+markdown fix in https://github.com/docker-library/docs/pull/2376

**Pull request closed without merging**